pvc_to_pvs_image_import/_topics/preparing_powervc_node.md

```diff
@@ -0,0 +1,223 @@
+
+
+2. Fill in the connection data and save the provider.
+
+
+3. Initiate the refreshing of the provider, wait for instance images to appear.
+
+
+### Add a PowerVC instance
+
+1.  Open PowerVC registration form using `Cloud Providers` Tab in ManageIQ.
+
+![PVC Registration Form Sample Screenshot](../images/pvc.png)
+
+2. Fill in the registration data.
+
+
+3.  Open `Image Import` Tab on the registration form.
+
+
+4.  Input the SSH password of the `root` user on the PowerVC node and save the provider.
+
+
+5.  Initiate the refreshing of the provider, wait for instance images to appear.
+
+
+### Add a Cloud Object Storage
+
+1.  Create new [Cloud Object Storage](https://cloud.ibm.com/objectstorage/create) instance in IBM Cloud Console.
+
+
+2.  Navigate to `Service Credentials -> New Credential`, choose the `Manager` role and 
+
+    enable `Include HMAC Credential` option when generating the credentials.
+
+
+3. Navigate to `Manage -> Access (IAM) -> Service IDs` and choose your newly created key. 
+
+
+4. Navigate to `Access Policies -> Assign Access -> IAM Service -> Cloud Object Storage`.
+
+
+5. Select `Writer` checkbox under `Service Access` group and press `Add`.
+
+![Access Policy Screenshot](../images/cos_access.png)
+
+6. Create a new bucket through IBM console, e.g. `transient-image-bucket`.
+
+
+![Create a new bucket Screenshot](../images/cos_buckets.png)
+
+7.  Open Cloud Object Storage registration form using `Storage Providers` Tab in ManageIQ.
+
+
+
+8.  Initiate the refreshing of the provider, wait for the cloud buckets to appear.
+
+### Start the workflow
+
+1. Navigate to the Power Virtual Servers provider added above.
+
+
+2. Click on the `Import Image` button.
+
+![Import Image Button Screenshot](../images/import_button.png)
+
+
+3. Choose the PowerVC provider added above as a source provider for the import.
+
+
+4. Choose the image you would like to import.
+
+
+5. Choose the Cloud Object Storage provider added above.
+
+
+6. Choose the transient bucket created above.
+
+
+7. Choose target disk type for the image being imported.
+
+
+8. Decide if you want to keep transient OVA file in your bucket upon completion.
+
+
+9. Submit the Image Import Request (it might take an hour or more depending on the image size).
+
+![Workflow Form Screenshot](../images/workflow.png)
+
+
+10. You can see current progress in `ManageIQ -> Settings -> Tasks -> All Tasks`.
+
+
+11.  Initiate the refreshing of the Power Virtual Servers provider upon workflow completion, 
+     
+     wait for the newly imported image to appear.
+
+
+### Provision an instance
+
+1. Navigate to Power Virtual Servers provider.
+
+
+2. Go to Images View and click on the newly imported image.
+
+
+3. Click on "`Lifecycle -> Provision Instances Using this Image`"
+
+
+4. Choose the server name and set all other values as desired. 
+
+
+5. Submit the provisioning workflow and wait for it to finish.
+
+
+6. Connect to the newly provisioned server e.g. via SSH.
```

Comment:
We should create a troubleshooting section somewhere through all of this in terms of where to look for logs, etc.